examples/arithmetics/package.json

```diff
@@ -77,8 +77,8 @@
         "langium-cli": "~2.1.0"
     },
     "volta": {
-        "node": "18.17.1",
-        "npm": "9.6.7"
+        "node": "18.18.2",
```

Comment:
Node `18.19.0` is already out 🙈 , but it contains npm v10 now (to my surprise).

Reply:
Yes, I know. I, however, don't wanna yell at users (like me) having 18.18.... installed. (18.18.0 was released mid of September '23)
It shouldn't make a difference for Langium.

Reply:
Alright